src/solution.c

```diff
@@ -54,6 +54,10 @@ systime_t last_dgnss;
 
 double soln_freq = 10.0;
 u32 obs_output_divisor = 2;
+u32 gpgsv_msg_period = 10;
```

Comment:
do we think "period: is clear as for the name of these parameters?  I don't want anyone to think that these are the "seconds" between messages.  It might make more sense to have them be called rates or output divisors.  Other mfctrs seem to call them rates or rate divisors.